cache-manager.php

```diff
@@ -252,6 +255,98 @@ function queue_post_purge( $post_id ) {
 		 */
 		$this->purge_urls = apply_filters( 'wpcom_vip_cache_purge_urls', $this->purge_urls, $post_id );
 	}
+
+	/**
+	 * Purge the cache for some terms
+	 *
+	 * Hooks the `clean_term_cache` action
+	 *
+	 * We do not respect requests to clear caches for the entire taxonomy,
+	 * as this would be potentially hundreds or thousands of PURGE requests.
+	 *
+	 * @param array  $ids            An array of term IDs.
+	 * @param string $taxonomy       Taxonomy slug.
+	 * @param bool   $clean_taxonomy Whether or not to clean taxonomy-wide caches
+	 */
+	function queue_term_purge( $ids, $taxonomy ) {
+		$get_term_args = array(
+			'taxonomy'    => $taxonomy,
+			'include'     => $ids,
+			'hide_empty'  => false,
+		);
+		$terms = get_terms( $get_term_args );
+		if ( is_wp_error( $terms ) ) {
+			return;
+		}
+		foreach ( $terms as $term ) {
+			if ( ! term_exists( $term->term_id, $taxonomy ) ) {
+				continue;
+			}
+			$this->queue_purge_urls_for_term( $term );
+		}
+	}
+
+	/**
+	 * Queue all URLs to be purged for a given term
+	 *
+	 * @param object $term A WP term object
+	 */
+	function queue_purge_urls_for_term( $term ) {
+
+		/**
+		 * Allows you to customise the URL suffix used to specify a page for
+		 * paged term archives.
+		 *
+		 * Developers should hook this filter to provide a different page
+		 * endpoint if they have custom or translated rewrite rules for
+		 * paging in term archives:
+		 *
+		 * Standard:     example.com/category/news/page/2
+		 * Non-standard: example.com/category/news/p/2
+		 *
+		 * The string should be formatted as for `sprintf`, with a `%d` in place
+		 * of the page number.
+		 *
+		 * @param string sprintf formatted string, including `%d`
+		 * }
+		 */
+		$paging_endpoint = apply_filters( 'wpcom_vip_cache_purge_urls_paging_endpoint', $GLOBALS['wp_rewrite']->pagination_base . '/%d/' );
+
+		/**
+		 * The maximum page to purge from each term archive when a post associated with
+		 * that term is published.
+		 *
+		 * e.g. if the value is 3, the following pagination URLs will be purged for the
+		 * news category archive:
+		 *
+		 * example.com/category/news/
+		 * example.com/category/news/page/2
+		 * example.com/category/news/page/3
+		 *
+		 * @param int The maximum page to purge from each term archive
+		 * }
+		 */
+		$max_pages = apply_filters( 'wpcom_vip_cache_purge_urls_max_pages', 5 );
```

Comment:
Yes, yes it should. Bad copy/pasta. Addressed in https://github.com/Automattic/vip-go-mu-plugins/pull/232/commits/973c32b67dbe6677e8641a6417037204e47fbefe

Thanks!